keps/0019-package-api-versioning.md

```diff
@@ -280,20 +313,50 @@ kudo install kafka --app-version 3.0.0
 
 Installs `kafka-3.0.0-1.1.0`.
 
+**for single version:**
+
+```
+kudo install kafka --app-version 2.3.0
+```
+
+Installs `kafka-1.1.0`.
+
+```
+kudo install kafka --app-version 3.0.0
+```
+
+Installs `kafka-2.2.1`.
+
 **Install a Kafka operator with a specific version**
 
 Let's say a user wants to use Apache Kafka `3.0.0` and even though there's a `3.0.0` with a `1.1.0` revision released, they want the operator with revision `1.0.0` due to reasons like:
 - revision `1.1.0` introduced a bug for which there's still no released fixes
 - revision `1.1.0` changed in a way that will require them to invest time and resources to adapt
 
+**for composite version:**
+
 ```
 kudo install kafka --version 3.0.0-1.0.0
 ```
 
+**for single version:**
+
+it should be enough with single version
+```
+kudo install kafka --version 2.2.0
```

Comment:
so is this reversing the order of which version has primacy? Or is this getting the latest version of Kafka and installing it at that operator version?

Reply:
it separates `--version` from `--app-version`